packages/acceptit-Core.package/ACUserStory.class/class/definition.st

```diff
@@ -7,6 +7,6 @@ definition
 			superklass := (super definition findTokens: ' ') first.
 			stream
 				nextPutAll: superklass; space;
-				nextPutAll: 'named:'; space; store: self storyName; crtab;
+				nextPutAll: 'named:'; space; store: self title; crtab;
```

Comment:
This change corrects the class name from the User stories, right?

Reply:
the naming was just inconsistent. The UserStory title was sometimes named title sometimes storyName. We renamed it to title everywhere.